frontend/packages/console-shared/src/components/editor/YAMLEditorToolbar.scss

```diff
@@ -0,0 +1,15 @@
+.ocs-yaml-editor-toolbar {
+  &__links {
+    display: flex;
```

Comment:
Add `flex-shrink: 0` to prevent this section from shrinking when small.
Saw this on safari:
![image](https://user-images.githubusercontent.com/14068621/77470593-50e0bc80-6de7-11ea-937a-43cf4e7d7f26.png)

Reply:
Done.